adafruit_macropad.py

```diff
@@ -324,6 +324,25 @@ def _keys_and_pixels(
             # No MIDI ports available.
             self._midi = None
 
+    @property
+    def display_sleep(self) -> bool:
+        """Whether the display is currently asleep."""
+        return self._display_sleep
+
+    @display_sleep.setter
+    def display_sleep(self, sleep: bool) -> None:
+        """Change the sleep status of the display.
+
+        If the display is put to sleep, it stops the OLED drive and greatly
+        reduces its power usage. The current content of the display is kept
+        in the memory and can still be updated.
+        """
```

Comment:
the docstring in the setter is ignored, it should all be in the getter docstring (in the docs, a `@property` is a single entry). Changing the first line maybe to "The current sleep status of the display, set it to put the display to sleep or wake it." etc..

```suggestion
    def display_sleep(self) -> bool:
        """
        The current sleep status of the display, set it to put the display to sleep or wake it.
        
        If the display is put to sleep, it stops the OLED drive and greatly
        reduces its power usage. The current content of the display is kept
        in the memory and can still be updated.
        """
        return self._display_sleep

    @display_sleep.setter
    def display_sleep(self, sleep: bool) -> None:
        """Change the sleep status of the display."""
```

Reply:
Thank you for the review. I didn't even know, that the docstring for the setter is ignored by sphinx.

I put the whole documentation into the setter now, added some more details from the datasheet and replaced the ternary.